### PR TITLE
Add instructions to import packages from PPA

### PR DESCRIPTION
**Pull request by j-rivero**
I've added instructions about how to import packages. Also clarify that we are using Focal Ubuntu's packages in PPA instructions.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -41,9 +41,12 @@ The first step will create a source package from DART git checkout in the local
 system and upload it to Open Robotics PPA.
 
 **One Ubuntu distribution needs to be released at a time**. The information
-about target distribution (start with bionic) goes only in the Changelog entry. To
+about target distribution (bionic) goes only in the Changelog entry. To
 release multiple distribution, repeat this step 1 changing the changelog entry 
-(focal is also supported) and uploading the new source package to the PPA.
+and uploading the new source package to the PPA.
```
Comment (azeey) [inline RELEASING.md]: If we are only doing bionic, do we need to "repeat this step ..."?

**Review by azeey — APPROVED** [review]
Looks good. I haven't tested it locally, but will follow instructions next time and update the documentation if there's anything missing. I left one minor comment.

File: RELEASING.md [hunk 2]
```diff
@@ -87,4 +90,20 @@ for the selected Ubuntu distribution in all arches (supported architectures can
...
 
-TODO
+### Update reprepro importer config
+
+Create a PR against the `refactor_osrfbuild` branch in reprepro-updater repository and change the 
+[SourceVersion field to the new one released in the PPA](https://github.com/ros-infrastructure/reprepro-updater/blob/refactor_osrfbuild/config/packages.osrfoundation.org/openrobotics_dart_packages_from_ppa.yaml#L8).
+
+### Run reprepro-importer Jenkins test
+
+Rebuild [this build](https://build.osrfoundation.org/job/reprepro_importer/33) of reprepro-importer
```
Comment (azeey) [inline RELEASING.md]: Build 33 is no longer available. Should we just say use the latest job or create a new job?

File: RELEASING.md [hunk 3]
```diff
@@ -87,4 +90,20 @@ for the selected Ubuntu distribution in all arches (supported architectures can
...
+### Update reprepro importer config
+
+Create a PR against the `refactor_osrfbuild` branch in reprepro-updater repository and change the 
+[SourceVersion field to the new one released in the PPA](https://github.com/ros-infrastructure/reprepro-updater/blob/refactor_osrfbuild/config/packages.osrfoundation.org/openrobotics_dart_packages_from_ppa.yaml#L8).
+
+### Run reprepro-importer Jenkins test
+
+Rebuild [this build](https://build.osrfoundation.org/job/reprepro_importer/33) of reprepro-importer 
+(note that the [paremeters](https://build.osrfoundation.org/job/reprepro_importer/33/parameters/) 
+set ubuntu_testing as repository which is a unofficial playground). Check the output to see the simulation
```
Comment (azeey) [inline RELEASING.md]: Looking at the jobs in https://build.osrfoundation.org/job/reprepro_importer/, it looks like they repository is set to either `ubuntu_stable` or `ubuntu_prerelease`. Can you verify that this should be `ubuntu_testing` in the text?

File: RELEASING.md [hunk 4]
```diff
@@ -41,9 +41,12 @@ The first step will create a source package from DART git checkout in the local
...
-about target distribution (start with bionic) goes only in the Changelog entry. To
+about target distribution (bionic) goes only in the Changelog entry. To
 release multiple distribution, repeat this step 1 changing the changelog entry 
-(focal is also supported) and uploading the new source package to the PPA.
+and uploading the new source package to the PPA.
+
+**Focal is not supported to be imported into packages.osrfoundation.org since we 
+are using Ubuntu's packages**
 
 A copy of [this fork](https://github.com/ignition-forks/dart) is required to be in the system.
```
Comment (azeey) [inline RELEASING.md:50]: I can't comment on unchanged lines, so this refers to line 61 below. The argument to `--new-version` should have the format `6.10.0-osrf<new deb version>-$(date +%Y%m%d)~$DISTRO~$(git rev-parse HEAD)`.  I called it "deb version", but you probably have a better name, but I couldn't upload to launchpad without updating that number to something greater than what's on launchpad at the time.

File: RELEASING.md (hunk 3 above)
Comment (j-rivero) [reply RELEASING.md]: `ubuntu_testing` is a playground repository, should work, yes.

File: RELEASING.md (hunk 2 above)
Comment (j-rivero) [reply RELEASING.md]: Mentioning the build was not a good idea, indeed. I just described the params here: dadb41e

File: RELEASING.md (hunk 1 above)
Comment (j-rivero) [reply RELEASING.md]: Simplified the whole thing in 65f7cbd

File: RELEASING.md (hunk 4 above)
Comment (j-rivero) [reply RELEASING.md:50]: It is not always mandatory to bump it (if not releasing twice in the same day) but it does not hurt anyway. 87d0a20

**Review by azeey — APPROVED** [review]

**Pull request merged**